bokeh/tile_providers.py

```diff
@@ -14,6 +14,17 @@
     )
 )
 
+#: Provide toner background tiles from http://tile.stamen.com
+STAMEN_TONER_BACKGROUND = WMTSTileSource(
+    url='http://tile.stamen.com/toner-background/{Z}/{X}/{Y}.png',
```

Comment:
The docs at [maps.stamen.com](http://maps.stamen.com) seem to indicate the url is: http://tile.stamen.com/toner/{z}/{x}/{y}.png